sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/src/AnalyzeConversationAuthoring.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Azure.Core;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.AI.Language.Conversations.Authoring;
+
+namespace Azure.AI.Language.Conversations.Authoring
+{
+    /// <summary> The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in &lt;a href=\"https://docs.microsoft.com/azure/cognitive-services/language-service/overview\"&gt;https://docs.microsoft.com/azure/cognitive-services/language-service/overview&lt;/a&gt;.0. </summary>
+    public partial class AnalyzeConversationAuthoring
```

Comment:
Looks like an empty class. Do you plan to update?

Reply:
I was thinking about adding specific constructors, now I see that it is not needed. Have deleted it now.